### PR TITLE
Support mock compute TargetSslProxy 

### PR DESCRIPTION
**Pull request by gemmahou**
### Change description

-Support mock compute TargetSslProxy
-Support mock regional SSLCertificate
-Add test case for compute forwarding rule with TargetSslProxy 
-Update Compute Address mock and update logs

### Tests you have done

- [x] Run `make ready-pr` to ensure this PR is ready for review.
- [x] Perform necessary E2E testing for changed resources.

### CODE REVIEW

File: pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargetsslproxy/_http.log [hunk 1]
```diff
@@ -0,0 +1,765 @@
...
+
+{
+  "certificate": "-----BEGIN CERTIFICATE-----\nMIIDJTCCAg0CFHdD3ZGYMCmF3O4PvMwsP5i8d/V0MA0GCSqGSIb3DQEBCwUAME8x\nCzAJBgNVBAYTAlVTMQswCQYDVQQIDAJXQTEhMB8GA1UECgwYSW50ZXJuZXQgV2lk\nZ2l0cyBQdHkgTHRkMRAwDgYDVQQDDAdFeGFtcGxlMB4XDTE5MDkyOTIyMjgyOVoX\nDTIwMDkyODIyMjgyOVowTzELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAldBMSEwHwYD\nVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQxEDAOBgNVBAMMB0V4YW1wbGUw\nggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDWLvOZIail12i6NXIqOspV\ncorkuS1Nl0ayrl0VuKHCvheun/s7lLLgEfifzRueYlSUtdGg4atWIwEKsbIE+AF9\nuUTzkq/t6zHxFAAWgVZ6/hW696jqcZX3yU+LCuHPLSN0ruqD6ZygnYDVciDmYwxe\n601xNfOOYRlm6dGRx6uTxGDZtfu8zsaNI0UxTugTp2x5cKB66SbgdlIJvc2Hb54a\n7qOsb9CIf+rrK2xUdJUj4ueUEIMxjnY2u/Dc71SgfBVn+yFfN9MHNdcTWPXEUClE\nFxd/MB3dGn7hVavXyvy3NT4tWhBgYBphfEUudDFej5MmVq56JOEQ2UtaQ+Imscud\nAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAMYTQyjVlo6TCYoyK6akjPX7vRiwCCAh\njqsEu3bZqwUreOhZgRAyEXrq68dtXwTbwdisQmnhpBeBQuX4WWeas9TiycZ13TA1\nZ+h518D9OVXjrNs7oE3QNFeTom807IW16YydlrZMLKO8mQg6/BXfSHbLwuQHSIYS\nJD+uOfnkr08ORBbLGgBKKpy7ngflIkdSrQPmCYmYlvoy+goMAEVi0K3Y1wVzAF4k\nO4v8f7GXkNarsFT1QM82JboVV5uwX+uDmi858WKDHYGv2Ypv6yy93vdV0Xt/IBj3\n95/RDisBzcL7Ynpl34AAr5MLm7yCSsPrAmgevX4BOtcVc4rSXj5rcoE=\n-----END CERTIFICATE-----\n",
```
Comment (yuwenma) [inline pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargetsslproxy/_http.log:259]: suggest change this sensitive data to fake data

**Review by yuwenma — COMMENTED** [review]
I spot-checked the golden log. It looks like there are still some diff we want to fix compared to the real GCP. I don't mind we do that in follow-up PRs. 
So once the secret is faked, I think this PR is good to go in.

File: pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargetsslproxy/_http.log (hunk 1 above)
Comment (gemmahou) [reply pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargetsslproxy/_http.log:259]: This is the test certificate we defined in [created.yaml](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/72072105ba81f39d5791e928b4625c04d919cc3f/pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargetsslproxy/dependencies.yaml#L41)

Comment (yuwenma) [reply pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargetsslproxy/_http.log:259]: Ah! That's wonderful! Too real to notice it is a fake! (FYI, sometimes people put fake at the beginning and the end of the CERTIFICATE to make it more obvious)

**Pull request merged**